src/plugins/intel_cpu/src/nodes/matmul.cpp

```diff
@@ -543,6 +543,17 @@ void MatMul::prepareParams() {
     if (!src0MemPtr || !src0MemPtr->isDefined() || !src1MemPtr || !src1MemPtr->isDefined())
         OPENVINO_THROW(errorPrefix, " has undefined input memory");
 
+    // check for degenerate case
```

Comment:
minor: would be nice to explain what is degenerate case in more details. Like in fact this is only one combination: non-zero batch, M and N dimension and zero K.

Reply:
Done